packages/manager/modules/pci/src/projects/project/users/download-openrc/translations/Messages_fr_FR.json

```diff
@@ -2,7 +2,7 @@
   "pci_projects_project_users_download-openrc_title": "Télécharger un fichier de configuration OpenStack",
   "pci_projects_project_users_download-openrc_submit_label": "Télécharger",
   "pci_projects_project_users_download-openrc_cancel_label": "Annuler",
-  "pci_projects_project_users_download-openrc_content": "Vous allez télécharger un fichier de configuration permettant de vous connecter à Openstack.",
+  "pci_projects_project_users_download-openrc_content": "Certains services comme l'Object Storage ou Cloud Archive sont localisés sur des régions dites 'globales' (comme GRA par exemple), la plupart des autres services sont sur des régions spécifiques (comme GRA5 par exemple).",
```

Comment:
Hmm ok for the modification of the text, but the key name seems not to be proper though

Reply:
You mean use of "-" ?

Reply:
I was wondering if the key name : `pci_projects_project_users_download-openrc_content` is still relevant, since the meaning of the text have changed